elytron/src/main/java/org/wildfly/extension/elytron/SSLDefinitions.java

```diff
@@ -991,4 +1003,28 @@ protected void executeRuntimeStep(OperationContext context, ModelNode operation)
         protected abstract ServiceUtil<SSLContext> getSSLContextServiceUtil();
     }
 
+    private static BooleanSupplier getFipsSupplier() {
+        try {
+            final Class<?> providerClazz = SSLDefinitions.class.getClassLoader().loadClass("com.sun.net.ssl.internal.ssl.Provider");
```

Comment:
I can't find such class in IBM distribution. So I assume that will throw ClassNotFoundException.